batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/controller/HTTPStatusHandler.java

```diff
@@ -0,0 +1,60 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.constants.*;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by sanjay on 1/19/16.
+ */
+public abstract class HTTPStatusHandler extends HttpServlet {
+  private static final Logger LOGGER = Logger.getLogger(HTTPStatusHandler.class.getName());
+
+
+  protected void show404(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+    response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+    request.setAttribute(Attribute.MESSAGE, Message.PAGE_NOT_FOUND);
```

Comment:
I don't think this whole class is needed at all. I would prefer handling errors from web.xml and leave container manage it.